extras/releases/azure/kustomization.yaml

```diff
@@ -0,0 +1,3 @@
+resources:
+- ..
```

Comment:
???

Reply:
Correct me if I'm wrong, but I'm pretty sure Kustomize prevents escaping the directory for security reasons.

Reply:
I have no idea what I'm doing. In my imagination I can simply include the repository from the parent path, but you would probably not ask if this would work.

Reply:
You can check by running `kustomize build ./` within this directory

Reply:
```
┗ ✨ kustomize build ./         
Error: accumulating resources: accumulation err='accumulating resources from '..': '~/Code/GiantSwarm/management-cluster-bases/extras/releases' must resolve to a file': cycle detected: candidate root '~/Code/GiantSwarm/management-cluster-bases/extras/releases' contains visited root '~/Code/GiantSwarm/management-cluster-bases/extras/releases/azure'
```

Reply:
Yeah, sure, I assumed that. So we will end up with including the repository and the according provider specific Kustomizations in our management cluster Kustomization then?

Reply:
You could do something like this:

```
resources:
- https://github.com/giantswarm/management-cluster-bases//extras/releases/?ref=nojrh
- releases-azure.yaml
```

(With the branch correctly set to main but that wont work until after merge)

Reply:
Ok, so for testing I'll refer to this branch and before merging I switch to `main`. Sounds good.